app/models/student.rb

```diff
@@ -1,2 +1,3 @@
 class Student < ActiveRecord::Base
+  belongs_to: :student
```

Comment:
Make sure that you always double-check your associations in the console to double-check that you didn't make any errors. The correct syntax for this is: ```belongs_to :teacher```